src/Commands/Taxonomy/MergeTerms.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.SharePoint.Client;
+using Microsoft.SharePoint.Client.Taxonomy;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PnP.PowerShell.Commands.Taxonomy
+{
+    [Cmdlet(VerbsData.Merge, "PnPTerms")]
```

Comment:
It should be `PnPTerm` . We cannot allow plural as per PowerShell convention.

Can you also update the same in docs ?

Reply:
Done - in both pluss file names.